HelpSource/Classes/Wavetable.schelp

```diff
@@ -95,8 +95,6 @@ argument::minval
 the minimum value in the plot. Defaults to the highest value in the data.
 argument::maxval
 the maximum value in the plot. Defaults to the lowest value in the data.
-argument::parent
-a window to place the plot in. If nil, one will be created for you.
```

Comment:
The warning thrown here is

> Method -plot has arg named 'separately', but doc has 'argument:: parent'

This deletion is fine but `separately` should be documented. Not sure if that's what you were asking.